.emdaer/README.emdaer.md

```diff
@@ -35,46 +35,53 @@
 -->
 
 <!--emdaer-p
-  - '@emdaer/plugin-node-package'
-  - path: '@emdaer/meta/lib/README/what-is-emdaer.js'
+  - '@emdaer/plugin-import'
```

Comment:
Remove

Reply:
?

Reply:
@flipactual github left the comment in the same spot, even though my commit resolved the issue. should be ok here.